abr_jaco2/interface/jaco2_rs485.cpp

```diff
@@ -596,11 +601,12 @@ void Jaco2::ProcessFeedback() {
                 PrintError(ii, current_motor);
                 // in case of an error, go on to the next packet
 
-                for (int jj = 0; jj < 6; jj++) {
-                    clear_error_message[jj].DataLong[1] =
-                        ((unsigned long) feedback_message[ii].DataLong[0]);
-                }
-                SendAndReceive(clear_error_message, false);
+                // preset to 0 to clear all errors
```

Comment:
this code is commented, should it be removed?

Reply:
removed